src/bundle/Resources/public/js/CKEditor/core/base-ckeditor.js

```diff
@@ -226,6 +231,29 @@ const VIEWPORT_TOP_OFFSET = 102;
                         textarea.dispatchEvent(new Event('input'));
                     }
                 };
+                const setDataSourceHeight = (toolbarNode, fieldEditNode) => {
+                    const dataSourceNode = fieldEditNode.querySelector('.ibexa-data-source');
+                    const toolbarNodeRect = toolbarNode.getBoundingClientRect();
+                    const dataSourceNodeRect = dataSourceNode.getBoundingClientRect();
+                    const { height: toolbarHeight } = toolbarNodeRect;
+                    const { top: dataSourceTop } = dataSourceNodeRect;
```

Comment:
I don't think these two extra variables are necessary here
```suggestion
                    const { height: toolbarHeight } = toolbarNode.getBoundingClientRect();
                    const { top: dataSourceTop } = dataSourceNode.getBoundingClientRect();
```